build.gradle

```diff
@@ -168,8 +168,8 @@ subprojects {
         googleCloudGaVersion = '1.65.0'
         log4j2Version = '2.11.1'
         signalfxVersion = '0.0.48'
-        springBootVersion = '1.5.15.RELEASE'
-        springBootTestVersion = '2.1.1.RELEASE'
+        springBootVersion = '2.1.5.RELEASE'
```

Comment:
IIRC Spring boot 1.x and 2.x are incompatible and we're still depending on 1.x in other artifacts. So maybe we need to keep the 1.5.15 version? /cc @dinooliva

Reply:
Yes - springBootVersion is used to build libraries.spring_boot_starter_web which is used to build spring_sleuth_v1x and that artifact is explicitly meant to be built with Spring 1.x (surprised the unit tests don't fail).

We may need to create a legacy version of the Spring boot library for use with existing artifacts and migrate them as possible.

Reply:
I have left the version as is for spring_sleuth_v1x. Changed the version for opencensus-contrib-spring.